src/rtnl/link/nlas/link_infos.rs

```diff
@@ -1749,6 +1859,31 @@ mod tests {
         assert_eq!(expected, parsed);
     }
 
+    #[rustfmt::skip]
+    #[test]
+    fn parse_info_bondport() {
+        let data = vec![
+            0x08, 0x00,                                        // length
+            0x04, 0x00,                                        // IFLA_INFO_SLAVE_KIND
+            0x62, 0x6f, 0x6e, 0x64,                            // "bond"
+
+            0x0c, 0x00,                 // length
+            0x05, 0x00,                 // IFLA_INFO_SLAVE_DATA
+                0x08, 0x00,             // length
+                0x09, 0x00,             // IFLA_BOND_SLAVE_PRIO
+                0x32, 0x00, 0x00, 0x00, // 50
```

Comment:
In realword, there is padding after 0x05. Please use:

```
sudo modprobe nlmon
sudo ip link add nlmon0 type nlmon
sudo ip link set nlmon0 up
```

This means you are guessing the binary data, not using real capture data here.

Reply:
![image](https://github.com/rust-netlink/netlink-packet-route/assets/36004580/ae3be4d0-3f6a-4375-92df-affa47a4405c)

I updated it, so it should match the Wireshark capture result.